change/@uifabric-example-app-base-2019-07-10-17-15-36-monaco-editor.json

```diff
@@ -0,0 +1,8 @@
+{
+  "comment": "Website changes to work with monaco editor state management",
+  "type": "minor",
+  "packageName": "@uifabric/example-app-base",
+  "email": "nishikaza@berkeley.edu",
```

Comment:
>nishikaza@berkeley.edu [](start = 12, length = 22)

use your ms alias please

Reply:
Auto-generated from change file, updated primary email in Github for future commits

---
In reply to: [302627750](https://github.com/OfficeDev/office-ui-fabric-react/pull/9769#discussion_r302627750) [](ancestors = 302627750)